frontend/javascript/components/docupload/image-page.vue

```diff
@@ -1,35 +1,146 @@
+<script setup>
```

Comment:
Low prio/later: We should probably improve this a bit. The "cut" button feels very unintuitive and it can't be undone without refreshing. Also the animation when rotating 360 degrees is very unsatisfying 😉